From bc34ba549d4366092a39a4d7bc4414f8cbd8bc6c Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sat, 3 Aug 2019 09:13:54 +0200
Subject: [PATCH] History.txt: use 10717 instead of xxxxx
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Set current development version to 4.10717 instead of 4.xxxxx to avoid
the following build failure when git is unavailable:

/home/fabrice/buildroot/output/build/domoticz-0f411f781ae4fb4a82f592d38a3f40578c149533/main/domoticz.cpp: In function ‘void GetAppVersion()’:
/home/fabrice/buildroot/output/build/domoticz-0f411f781ae4fb4a82f592d38a3f40578c149533/main/../appversion.h:1:20: error: ‘xxxxx’ was not declared in this scope
 #define APPVERSION xxxxx

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 History.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/History.txt b/History.txt
index 4dc45e6b5..ad67976f3 100644
--- a/History.txt
+++ b/History.txt
@@ -1,4 +1,4 @@
-Version 4.xxxxx (xxx 2019)
+Version 4.10717 (xxx 2019)
 - Implemented: DarkSky CloudCover sensor
 - Implemented: EventSystem, option in Settings to enable/disable URL call logging containing full URL path
 - Implemented: GUI, OpenZWave added Refresh Node Information button, styling
-- 
2.20.1

